src/OrchardCore.Modules/OrchardCore.Menu/README.md

```diff
@@ -20,6 +20,10 @@ The `Menu` shape is used to render a Menu.
 
 #### Menu Example
 
+```cshtml
```

Comment:
Create another section like `#### Rendering a menu` where you would paste this block.

Reply:
I removed this section after learning there is no direct relationship with menu name.